### PR TITLE
ES6 convert formstack-iframe-embed

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

Convert formstack-iframe-embed.js to ES6

## What is the value of this and can you measure success?

More ES6!

## Does this affect other platforms - Amp, Apps, etc?

No

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No

## Screenshots

N/A

## Tested in CODE?

No

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/identity/formstack-iframe-embed.js [hunk 1]
```diff
@@ -0,0 +1,295 @@
+// @flow
+
+/**
+    Formstack - composer integration
+
+    This script runs INSIDE a formstack iframe and is initialised by the NGW
+    requireJs setup
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/identity/formstack-iframe-embed.js]: I guess this is no longer initialised by the NGW (Next Gen Web?) RequireJS setup 😄

Comment (GHaberis) [reply static/src/javascripts/projects/common/modules/identity/formstack-iframe-embed.js]: Hah, yeah I'll bring this comment up-to-date!

File: static/src/javascripts/projects/common/modules/identity/formstack-iframe-embed.js [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+
+            // if no errors, submit form
+            if (fsFormErrors.length === 0) {
+                this.form.submit();
+            }
+        }, 100);
+    }
+}
+
+export default Formstack;
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/identity/formstack-iframe-embed.js]: Shall we use a named export?

**Review by SiAdcock — APPROVED** [review]
Dinky comments, otherwise 🚀

**Pull request merged**